frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/tests/EmbeddingDataPicker.unit.spec.tsx

```diff
@@ -76,16 +95,20 @@ describe("EmbeddingDataPicker", () => {
         expect(screen.queryByText("Saved Questions")).not.toBeInTheDocument();
       });
 
-      it('should show both models and tables when `entity_types=["models", "table"]`', async () => {
-        setup({
-          entityTypes: ["model", "table"],
-        });
+      it.each([2, 3])(
+        'should show both models and tables when `entity_types=["models", "table"] with %i models',
```

Comment:
Ensure that the user's provided `entity_types` always take precedence over the default behavior of hiding tables.